icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/errorprone/ErrProneUtils.java

```diff
@@ -0,0 +1,92 @@
+//·©·2025·and·later:·Unicode,·Inc.·and·others.
+
+package com.ibm.icu.dev.tool.errorprone;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableBiMap;
+import com.google.errorprone.BugCheckerInfo;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.scanner.BuiltInCheckerSuppliers;
+
+class ErrProneUtils {
+
+    /*
+     * This is from errorprone library, a list of all known issue types.
+     *
+     * The reason why we need all of this complication:
+     *
+     * The errorprone tool is implemented as a javac compiler extension.
+     * So when the code is compiled, the errorprone "hooks" are invoked by the
+     * Java compiler.
+     * As such, errorprone can only use the Java compiler mechanisms to report issues.
+     * But if it reports and error, the compiler stops compiling the rest of the files.
+     * In order to generate a report on the whole source tree errorprone has a flag
+     * to report all issues as warnings (-XepAllErrorsAsWarnings).
+     * That way the compiler continues on the whole project and does not stop.
+     * But because of that even the worst errorprone violations, that would qualify
+     * as errors, are still reported as warnings.
```

Comment:
```suggestion
     * But because of that, even the worst errorprone violations that would qualify
     * as errors are still reported as warnings.
```

Reply:
not done yet